massgov/mayflower
# Patternlab/DP-16073:  Fine tune campaign page styling

___DESCRIPTION___
Fixed
- (Patternlab) [MarketingCampaign] DP-16073: Fix spacing between marketing campaign page sections. #772
- (Patternlab) [Card] DP-16073: Stack image on primary card on tablet size screen. #772
- (Patternlab) [CampaignVideo] DP-16073: Remove heading style override and wrapper top/bottom padding in preference of inheriting margins from page. #772

Testing:
1. Go to http://localhost:3000/?p=pages-marketing-campaign
2. Confirm header/spacing between sections is appropriate / default mayflower
2. Confirm primary card image/content stacks at medium breakpoint (780px) vs small as before
